src/core/annotation.js

```diff
@@ -378,7 +370,9 @@ var Annotation = (function AnnotationClosure() {
       }
       opList.addOp(OPS.endAnnotations, []);
       annotationsReadyCapability.resolve();
-    }, reject);
+    }, function(reason) {
+      annotationsReadyCapability.reject(reason);
+    });
```

Comment:
Would it be possible to simplify this even further, by removing the `PromiseCapability` all together? E.g. like this:
```javascrip
return Promise.all(annotationPromises).then(function(datas) {
  opList.addOp(OPS.beginAnnotations, []);
  for (var i = 0, n = datas.length; i < n; ++i) {
    var annotOpList = datas[i];
    opList.addOpList(annotOpList);
  }
  opList.addOp(OPS.endAnnotations, []);
});  
```